packages/syft/src/syft/service/code/user_code.py

```diff
@@ -1065,7 +1066,24 @@ def to_str(arg: Any) -> str:
         exec(code_item.parsed_code, _globals, locals())  # nosec
 
         evil_string = f"{code_item.unique_func_name}(**kwargs)"
-        result = eval(evil_string, _globals, _locals)  # nosec
+        try:
+            result = eval(evil_string, _globals, _locals)  # nosec
+        except Exception:
+            if context.job is not None:
+                # stdlib
+                import traceback
+
+                time = datetime.datetime.now().strftime("%d/%m/%y %H:%M:%S")
+                error_msg = traceback.format_exc()
+                __builtin__.print(
+                    f"{time} EXCEPTION LOG ({job_id}):", error_msg, file=sys.stderr
+                )
```

Comment:
Solves: `Error is printed to stdout such that we can see it in the logs of the server side`